paderbox/array/interval/core.py

```diff
@@ -358,6 +358,42 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
+    def as_serializable(self):
+        """
+        Exports intervals and length of `ArrayInterval` to a serializable object.
+        Intervals are exported as self._intervals_to_str to be human readable. 
+        Allows easy export of ArrayIntervals, e.g. into .json format. 
+        >>> from IPython.lib.pretty import pprint
+        >>> from paderbox.array.interval.core import ArrayInterval
+        >>> ai = ArrayInterval.from_str('1:4, 5:20, 21:25', shape=50)
+        >>> ai
+        ArrayInterval("1:4, 5:20, 21:25", shape=(50,))
+        >>> ai.as_serializable
+        ('1:4, 5:20, 21:25', (50,))
+        """
+        assert self.inverse_mode is False, 'Export of intervals as tuple is only valid for normal mode, not inverse mode!' 
+        return self._intervals_as_str, self.shape
+
+    @staticmethod
+    def from_serializable(obj):
+        """
+        Reverts `as_serializable`.
+        Args:
+            array: Object of length 2 with items (str: intervals, shape)
+        Returns:
```

Comment:
The Google docstring format recommends newlines between the sections like "Args" and "Returns". It also recommends an "Examples" section for the examples/doctests, but not everyone likes this.

Reply:
And rename "array" -> "obj" in docstring